DOCS-1826: Link between generics (#2507)
viamrobotics/docs: docs/registry/advanced/generic/_index.md

However, if you want to create a new service to monitor your machine's CPU and RAM usage for example, you need very different functionality that isn't currently exposed in any API.
Instead, you can use the generic service API to add support for your unique type of service, like local system monitoring, to your machine.

Use generic for a {{< glossary_tooltip term_id="modular-resource" text="modular resource" >}} model that represents a unique type of service.

> If your use case calls for adding support to unique or proprietary hardware, rather than adding new high-level software functionality, consider using the [generic component](/components/generic/) instead.

Reviewer: ```suggestion
If you are adding support for unique or proprietary hardware, rather than adding new high-level software functionality, use the [generic component](/components/generic/) instead.
```
[opt] For brevity, and then they really should use component so can we just say so?
